### PR TITLE
Clean up first

### PR DESCRIPTION
**Pull request by BKSteve**
There's still loads to delete and clean out. Just did language typos.

### CODE REVIEW

File: Docker.md [hunk 1]
```diff
@@ -28,7 +29,8 @@ services:
 
 #### Docker run
 
-```bash
```
Comment (miigotu) [inline Docker.md:31]: This should stay this way. It is how you tell GitHub flavored markdown what language to highlight the code in the code block as.

File: NZBtoMedia.md [hunk 2]
```diff
@@ -28,7 +28,7 @@ For more info see [NZBtoMedia's Repo](https://github.com/clinton-hall/nzbToMedia
 ## How to setup NZBtoMedia with NZBget.?
 
 The good news is that NZBget has NZBtoMedia support build-in to the program. This allows for easy configuration.  
-First you need to make sure that NZBget points to the NZBtoMedia files. Sometimes the files are included in the NZBget/Scrips folder. However if they are not you have two options. Manually download the latest NZBtoMedia [package](https://github.com/clinton-hall/nzbToMedia/archive/master.zip) and unpack them there. Or point to the NZBtoMedia folder included within SickChill. This can be done in NZBGet by going to Settings --> PATHS --> `ScriptDir`.
+First you need to make sure that NZBget points to the NZBtoMedia files. Sometimes the files are included in the NZBget/Scrips folder. However, if they are not you have two options. Manually download the latest NZBtoMedia [package](https://github.com/clinton-hall/nzbToMedia/archive/master.zip) and unpack them there. Or point to the NZBtoMedia folder included within SickChill. This can be done in NZBGet by going to Settings --> PATHS --> `ScriptDir`.
```
Comment (miigotu) [inline NZBtoMedia.md:31]: Nzbtomedia is no longer included directly in SC

**Review by miigotu — COMMENTED** [review]
I didn't have time to read all of the wiki data here, but there word changes look good. Noticed two small information problems in the wiki when scanning through

File: NZBtoMedia.md (hunk 2 above)
Comment (BKSteve) [reply NZBtoMedia.md:31]: This I need to rely on others, I've no idea except adjusting text and layout.

File: Docker.md (hunk 1 above)
Comment (BKSteve) [reply Docker.md:31]: before the \```bash line disappeared so I adjusted to the `bash` was visible.
change noted.

**Pull request merged**